sass/styles.scss

```diff
@@ -0,0 +1,113 @@
+$font-main: 'Lato', sans-serif;
```

Comment:
Overall you've implemented Sass, as a file type and used a few variables but you can push this further by making use of nesting and looking for chances to use mixins. I'll highlight some places for this as they arise.

Reply:
Another way to user variables is to replace color values with named color variables. You have at least two colors I found on a quick scan that you could replace with variables.